src/main/java/de/tum/in/www1/artemis/service/connectors/localci/buildagent/LocalCIDockerService.java

```diff
@@ -66,6 +67,11 @@ public class LocalCIDockerService {
     @Value("${artemis.continuous-integration.container-cleanup.cleanup-schedule-minutes:60}")
     private int containerCleanupScheduleMinutes;
 
+    // The image architecture that is supported by the build agent
+    // amd64 is the default value, as this is the architecture of Intel and AMD CPUs, which most systems still use
+    @Value("${artemis.continuous-integration.image-architecture:amd64}")
```

Comment:
Would it be possible to use the built-in
```java
System.getProperty("os.arch");
```
instead? Then it would not have to be configured in Artemis and always match the actual CPU architecture of the build agent.